### PR TITLE
Edit Data Migration Docs

### PR DESCRIPTION
**Pull request by erinysong**
## Changes

- Reformats data migration docs for readability and clarity.

#### Satisfied acceptance criteria and met development standards

- [ ] Met the acceptance criteria, or will meet them in a subsequent PR
- [ ] Created/modified automated tests
- [ ] Added at least 2 developers as PR reviewers (only 1 will need to approve)
- [ ] Messaged on Slack or in standup to notify the team that a PR is ready for review
- [ ] Changes to “how we do things” are documented in READMEs and or onboarding guide
- [ ] If any model was updated to modify/add/delete columns, makemigrations was ran and the assoicated migrations file has been commited.

#### Ensured code standards are met (Original Developer)

- [ ] All new functions and methods are commented using plain language
- [ ] Did dependency updates in Pipfile also get changed in requirements.txt?
- [ ] Interactions with external systems are wrapped in try/except
- [ ] Error handling exists for unusual or missing values

#### Validated user-facing changes (if applicable)

- [ ] New pages have been added to .pa11yci file so that they will be tested with our automated accessibility testing
- [ ] Checked keyboard navigability
- [ ] Tested general usability, landmarks, page header structure, and links with a screen reader (such as Voiceover or ANDI)
- [ ] Add at least 1 designer as PR reviewer

#### Reviewed, tested, and left feedback about the changes

- [ ] Pulled this branch locally and tested it
- [ ] Reviewed this code and left comments
- [ ] Checked that all code is adequately covered by tests
- [ ] Made it clear which comments need to be addressed before this work is merged
- [ ] If any model was updated to modify/add/delete columns, makemigrations was ran and the assoicated migrations file has been commited.

#### Ensured code standards are met (Code reviewer)

- [ ] All new functions and methods are commented using plain language
- [ ] Interactions with external systems are wrapped in try/except
- [ ] Error handling exists for unusual or missing values
- [ ] (Rarely needed) Did dependency updates in Pipfile also get changed in requirements.txt?

#### Validated user-facing changes as a developer

- [ ] New pages have been added to .pa11yci file so that they will be tested with our automated accessibility testing
- [ ] Checked keyboard navigability
- [ ] Meets all designs and user flows provided by design/product
- [ ] Tested general usability, landmarks, page header structure, and links with a screen reader (such as Voiceover or ANDI)

- [ ] Tested with multiple browsers, the suggestion is to use ones that the developer didn't (check off which ones were used)
  - [ ] Chrome
  - [ ] Microsoft Edge
  - [ ] FireFox
  - [ ] Safari

- [ ] (Rarely needed) Tested as both an analyst and applicant user

**Note:** Multiple code reviewers can share the checklists above, a second reviewers should not make a duplicate checklist

#### Verified that the changes match the design intention

- [ ] Checked that the design translated visually
- [ ] Checked behavior
- [ ] Checked different states (empty, one, some, error)
- [ ] Checked for landmarks, page heading structure, and links
- [ ] Tried to break the intended flow

#### Validated user-facing changes as a designer

- [ ] Checked keyboard navigability
- [ ] Tested general usability, landmarks, page header structure, and links with a screen reader (such as Voiceover or ANDI)

- [ ] Tested with multiple browsers (check off which ones were used)
  - [ ] Chrome
  - [ ] Microsoft Edge
  - [ ] FireFox
  - [ ] Safari

- [ ] (Rarely needed) Tested as both an analyst and applicant user

### CODE REVIEW

File: docs/operations/data_migration.md [hunk 1]
```diff
@@ -78,23 +78,24 @@ An example script using this technique is in
...
+|1| **escrow_domain_contacts.daily.gov.GOV.txt**  | Has the map of domain names to contact ID. Domains in this file will usually have 3 contacts each
+|2| **escrow_contacts.daily.gov.GOV.txt**         | Has the mapping of contact id to contact email address (which is what we care about for sending domain invitations)
+|3| **escrow_domain_statuses.daily.gov.GOV.txt**  | Has the map of domains and their statuses
+|4| **escrow_domains.daily.dotgov.GOV.txt**       | Has a map of domainname, expiration and creation dates
+|5| **domainadditionaldatalink.adhoc.dotgov.txt** | Has the map of domains to other data like authority, organization, & domain type
+|6| **domaintypes.adhoc.dotgov.txt**              | Has data on federal type and organization type
+|7| **organization.adhoc.dotgov.txt**             | Has organization name data
+|8| **authority.adhoc.dotgov.txt**                | Has authority data which maps to an agency
+|9| **agency.adhoc.dotgov.txt**                   | Has federal agency data
+|10| **migrationFilepaths.json**                  | A JSON which points towards all given filenames. Specified below.
```
Comment (therealslimhsiehdy) [inline docs/operations/data_migration.md:96]: omg bless you for adding this formatting

File: docs/operations/data_migration.md [hunk 2]
```diff
@@ -119,21 +120,22 @@ Later on, we will bundle this file along with the others into its own folder. Ke
 We need to run a few scripts to parse these files into our domain tables.
 We can do this both locally and in a sandbox.
 
-#### STEP 3: Bundle all relevant data files into an archive
+#### Step 3: Bundle all relevant data files into an archive
```
Comment (CocoByte) [inline docs/operations/data_migration.md:123]: Change to three ### for consistency?

**Review by CocoByte — COMMENTED** [review]
LGTM -- I think the changes do really help clarify how to setup and run migrations.  Just one, tiny non-blocking comment

File: docs/operations/data_migration.md (hunk 2 above)
Comment (erinysong) [reply docs/operations/data_migration.md:123]: Thank you for catching that!

**Review by zandercymatics — APPROVED** [review]
LGTM. Love the work you did on this, helps immensely

File: docs/operations/data_migration.md [hunk 3]
```diff
@@ -119,21 +120,22 @@ Later on, we will bundle this file along with the others into its own folder. Ke
...
 
 **WARNING:** All files uploaded in this manner are temporary, i.e. they will be deleted when the app is restaged.
 Do not use these environments to store data you want to keep around permanently. We don't want sensitive data to be accidentally present in our application environments.
 
-#### STEP 1: Using cat to transfer data to sandboxes
+### Step 1: Transfer data to sandboxes
+Use the following cat command to upload your data to a sandbox environment of your choice:
 
 ```bash
 cat {LOCAL_PATH_TO_FILE} | cf ssh {APP_NAME_IN_ENVIRONMENT} -c "cat > /home/vcap/tmp/{DESIRED_NAME_OF_FILE}"
```
Comment (therealslimhsiehdy) [inline docs/operations/data_migration.md:141]: **Question:** Dumb question, but is there a command for sending not just each file separately, but a whole folder of files at once? Asking as when I was doing this, I was trying to send full folders at once (the tar.gz) and not just each file separately.

ie 
Would `LOCAL_PATH_TO_FILE` be `src/tmp/folder_name_here` and `DESIRED_NAME_OF_FILE` be `folder_name_here`?

Comment (erinysong) [reply docs/operations/data_migration.md:141]: oops did not see this before merging I'm sorry. I believe generally the file would be a .tar.gz zip file that once unzipped contains multiple files and afterwards the next steps would detail how to unzip and add the migration updates. I believe there is also instructions for if those migrations are via .txt file though I personally never used this method

Comment (therealslimhsiehdy) [reply docs/operations/data_migration.md:141]: Haha you're okay! I'll ping Zander about it post Thanksgiving.... when I tried the SCP method below I couldn't get the folders or the tar files to send so I've been like ????? if there was something missing 🤣 If I figure it out I'll comment the command I use!

**Pull request merged**